CMakeLists.txt

```diff
@@ -201,7 +201,10 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
 	add_compile_options(-mwindows)
 	set(WIN_LINK_FLAGS "-Wl,--export-all-symbols -Wl,--subsystem,windows")
 else()
-	add_link_options(-rdynamic)
+	# TO DO: find a way to set -rdynamic in cmake < 3.13
+	if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13")
+		add_link_options(-rdynamic)
+	endif()
```

Comment:
Hmm, can we just move `cmake_minimum_required()`? I'm already getting these for some time:

```
CMake Deprecation Warning at CMakeLists.txt:18 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.
```

I mean,  ubuntu 18.04 is using 3.10.2 which is still under 3.13 but hmm, do we still want to support 18.04 :thinking: ?

So, I'm tempted to say to just move to 3.13 at least....

Reply:
The end of life for Ubuntu 18.04 LTS is April 30, 2023 which is the near future. So it would make sense to drop support.

Reply:
I'm +1 in dropping support and just move to 3.13 (or whatever is used in Ubuntu 20.04; I'm stating Ubuntu because they typically have fairly old releases). Otherwise, we need to provide already a full solution or people running < 3.13 will have problems anyways (just in runtime vs compile time).

Reply:
currently, the old version of osc, using gtk2, still builds and works fine on ubuntu 20, which comes with cmake 3.16. if we no longer want to support ubuntu 18, we can bump the required cmake version to 3.13, as long as the cmake version that comes on kuiper is also greater than/equal to 3.13

Reply:
Debian 10 "Buster" ships CMake 3.13, so that would be a good candidate.